### ISSUE TITLE
Add ability for Sue to send images

### ISSUE THREAD

**Issue by Manwholikespie** [inculi/Sue#3]
I tried working on this a while back, but didn't get very far. I think the important thing is having a definite file structure. Now that I know about uuid's, it shouldn't be too big of a problem. I definitely want to have things running more async by then, though. It would probably require sending messages directly...

Anyways, I wanted to post [this link](https://forums.macrumors.com/threads/send-image-from-messages-app-to-imessage.1471616/) here as a reminder for myself. Here's the code just incase:
```applescript
set theAttachment1 to POSIX file /Users/robert/pic.png
tell application "Messages"
    send theAttachment1 to buddy "+1 (xxx) x7 " of service "E:yourappleid@gmail.com"
end tell
```

**Comment by Manwholikespie** [comment]
I've been trying to get images to work in signal for some time. I think a big part of it is that Java on OS X can't infer mime-types properly. I've found that creating a `~/mime.types` (or maybe it's `~/.mime.types`, idk I made both and I don't know which one fixed it) will get it to stop returning null for the mimetype.

However, for some reason, after re-compiling signal-cli, I still can't get it to work. I'm thinking it might be my JDK version. I'll keep looking. In the meantime, I've forked the fork of signal-cli that Sue runs on and I've been tinkering with it.
